config/manager/kustomization.yaml

```diff
@@ -4,4 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: IMAGE_URL
+  newName: controller
+  newTag: 20200228-150928
```

Comment:
Don't think this change should be committed, placeholder of `IMAGE_URL` should probably remain or have I misunderstood?

Reply:
You're right!